src/colorize.cr

```diff
@@ -350,7 +307,7 @@ struct Colorize::Object(T)
     self
   end
 
-  def on(color : Symbol)
```

Comment:
should there be a deprecation warning for this?

Reply:
As mentioned in the OP, the mere existence of a `Symbol`-accepting overload will trigger deprecations even for arguments that autocast to `Color` due to overload ordering.